docs/tests/integration/test_fim/test_files/test_basic_usage/test_basic_usage_realtime_unsupported.md

```diff
@@ -0,0 +1,52 @@
+# Test basic usage realtime unsupported
+
+The test check for the correct transition to FIM mode scheduled when realtime is not avaible in the current OS and is
+set in the ossec.conf as a monitorization mode.
+
+## General info
+
+| Tier | Platforms | Time spent | Test file |
+|:--:|:--:|:--:|:--:|
+| 0 | Macos | 65 | test_basic_usage_realtime_unsupported.py
+| 0 | Solaris | 64 | test_basic_usage_realtime_unsupported.py
```

Comment:
I believe the time spent column should be in `hh:mm:ss` format.

Reply:
From other reviews  I was told to use ss format only